keel-sql/src/main/resources/db/changelog/20200325-create-artifact-last-checked-table.yml

```diff
@@ -0,0 +1,38 @@
+databaseChangeLog:
+- changeSet:
+    id: create-artifact-last-checked-table
+    author: fletch
+    changes:
+    - createTable:
+        tableName: artifact_last_checked
+        columns:
+        - column:
+            name: artifact_uid
+            type: char(26)
+            constraints:
+              unique: true
+              nullable: false
+        - column:
+            name: at
+            type: timestamp(3)
+            # MySQL is stupid and won't let you insert a zero valued TIMESTAMP
+            defaultValueDate: "1970-01-01T00:00:01"
+            constraints:
+              nullable: false
```

Comment:
A "last checked" table for artifacts just like we have for resources & delivery configs / environments